src/models/auth.js

```diff
@@ -10,7 +10,7 @@ const POLICIES = {
   MANAGER_USER: 'manager_user',
   LIST_TREE: 'list_tree',
   APPROVE_TREE: 'approve_tree',
-  LIST_PLANTER: 'list_planter',
+  LIST_GROWER: 'list_GROWER',
```

Comment:
The key could change to `LIST_GROWER` but the value will need to remain as `list_planter` for compatibility with the production database.